### PR TITLE
refactor: update notification footer styles

### PR DESCRIPTION
**Pull request by setchy**
handful of small tailwind tweaks

- prevent avatar from being shrunk on overflow
- allow horizontal scrolling on overflow (for notifications with lots of pills. This is a little quirky, so open to suggestions on a better method)
- simplify margins by using gap

### CODE REVIEW

**Review by afonsojramos — APPROVED** [review]

**Comment by afonsojramos** [comment]
Before merging, do you have an example of an overflow? Might be worth looking over

**Comment by setchy** [comment]
> Before merging, do you have an example of an overflow? Might be worth looking over

https://github.com/gitify-app/gitify/assets/386277/ac96bf70-c6de-4be8-bae6-3f88dbb199b2

**Comment by setchy** [comment]
would be nice to use less space for the update type and formatted datetime... but couldn't think of anything clean yet...

**Comment by bmulholland** [comment]
I don't think I'd ever notice that horizontal scroll myself. Wrapping to new line isn't an option?

**Review by bmulholland — APPROVED** [review]
Regardless of UX, this code is a strict improvement and is good to go, IMO. Other changes discussed could/should be a separate step, I think.

**Comment by setchy** [comment]
> Regardless of UX, I believe this code is a strict improvement and is good to go. IMHO.,other changes discussed above could/should be a separate step.

Thanks @bmulholland. I think this is pragmatic and affords us more time to find alternative solutions without holding up the next release

**Pull request merged**
